Make.inc

```diff
@@ -177,7 +177,7 @@ endif
 USE_GPL_LIBS ?= 1
 
 # Directories where said libraries get installed to
-prefix ?= $(abspath julia-$(JULIA_COMMIT))
+prefix ?= $(BUILDROOT)/julia-$(JULIA_COMMIT)
```

Comment:
@vtjnash can you think of any reason not to make this change? Part of the problem here was when `examples/Makefile` included `Make.inc`, the `prefix` was getting set to `examples/julia-$(JULIA_COMMIT)` which of course nothing else built into